### PR TITLE
Blacklist scala devel stuff

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

**Review by glitsj16 — APPROVED** [review]
LGTM

File: etc/inc/disable-programs.inc [hunk 1]
```diff
@@ -466,6 +467,7 @@ blacklist ${HOME}/.config/iridium
 blacklist ${HOME}/.config/itch
 blacklist ${HOME}/.config/jami
 blacklist ${HOME}/.config/jd-gui.cfg
+blacklist ${HOME}/.config/jgit
```
Comment (kmk3) [inline etc/inc/disable-programs.inc:470]: AFAIK jgit is the generic git implementation used by Java programs, so this
could possibly affect programs like Eclipse and SonarQube.

File: etc/inc/disable-programs.inc [hunk 2]
```diff
@@ -714,6 +717,7 @@ blacklist ${HOME}/.icedove
 blacklist ${HOME}/.imagej
 blacklist ${HOME}/.inkscape
 blacklist ${HOME}/.itch
+blacklist ${HOME}/.ivy2
```
Comment (kmk3) [inline etc/inc/disable-programs.inc:720]: This is used by Apache Ivy (and maybe also maven and gradle) and so could
potentially impact any Java development tool (including IDEs).

**Review by kmk3 — COMMENTED** [review]
This seems to affect more than just Scala.

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**